ckan/config/middleware/flask_app.py

```diff
@@ -341,3 +364,19 @@ def is_blueprint(mm):
         for blueprint in inspect.getmembers(module, is_blueprint):
             app.register_blueprint(blueprint[1])
             log.debug(u'Registered core blueprint: {0!r}'.format(blueprint[0]))
+            # print app.url_map
+            # building the routes and saving to config
+            # need to be improved
+            for rule in app.url_map.iter_rules():
```

Comment:
i think we have done this to get the routes in same place, because i had issues with the routes 
https://github.com/ckan/ckan/blob/3229-api-blueprint/ckan/lib/helpers.py#L869 so this looks like good solution. 

Maybe we could do this after core and extension blueprints are registered